src/main/webapp/consentGroup/NewConsentGroup.js

```diff
@@ -127,13 +124,11 @@ class NewConsentGroup extends Component {
         this.state.user.displayName,
         this.state.user.userName)
         .then(resp => {
-          // TODO: window.location.href is a temporal way to redirect the user to project's consent-group page tab. We need to change this after
-          // transitioning from old gsps style is solved.
-          window.location.href = [component.serverURL, "project", "main?projectKey=" + component.projectKey + "&tab=consent-groups&new"].join("/");
-          spinnerService.hide(CONSENT_SPINNER);
+          this.props.history.push('/project/main?projectKey=' + qs.parse(this.props.location.search).projectKey + '&tab=consent-groups&new', {tab: 'consent-groups'});
```

Comment:
This is nice, but technically not related to the main thrust of the PR. In the future, we should strive to keep them separate. I'm fine with this for this PR though.